UKHomeOffice/nmsw-ui
# NMSW-177 Register confirmation page (verify your email page)

# Ticket

NMSW-177

----

## AC

Given the app has requested account creation
When a success response is received
Then I am shown the confirmation page

----

## To test

- run app
- go to create account
- add a new email address and submit
- > you should be taken to the check your email page (design matches prototype)
- > your email address should be shown on this page
- click the link to resend your email
- > you should be taken to the resend page (no design/content yet)
- > your email address should be shown on this page

----

## Notes

- sending of verification email  is not yet available, see NMSW-183
